protocol/scripts/deploys/mainnet/v2/post-deploy/temple-v2-transactions-batch.json

```diff
@@ -0,0 +1,470 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1690182680732,
+  "meta": {
+    "name": "Temple V2 & TLC",
+    "description": "",
+    "txBuilderVersion": "1.16.1",
+    "createdFromSafeADDRESS": "0x4D6175d58C5AceEf30F546C0d5A557efFa53A950",
+    "createdFromOwnerADDRESS": "",
+    "checksum": "0x34fd3c2dab040576a17db549ea9b158fd9836d0ecec4245c24082e50ef27e70e"
+  },
+  "transactions": [
+    {
+      "to": "0x470EBf5f030Ed85Fc1ed4C2d36B9DD02e77CF1b7",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          { "internalType": "ADDRESS", "name": "account", "type": "ADDRESS" }
+        ],
+        "name": "addMinter",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "account": "0xXXXX" // TLC base strategy.ADDRESS
+      }
+    },
+    {
```

Comment:
We shouldn't need addMinter at all for mainnet.

So just two files needed:
 - acceptExecutor's (where the safe address is unknown still - so don't say it's the `0x4D61....` msig just in case we forget)
 - acceptRescuer's (where the safe address is unknown still - so don't say it's the `0x4D61....` msig just in case we forget)